times.go

```diff
@@ -25,11 +25,40 @@ func Time(then time.Time) string {
 	return RelTime(then, time.Now(), "ago", "from now")
 }
 
-var magnitudes = []struct {
+// FormatRelTime allows users to customize units and output format for relative time
+func FormatRelTime(then time.Time, format CustomRelFormat) string {
```

Comment:
The other two `Format*` funcs have these signatures:

```Go
func FormatFloat(format string, n float64) string {}
func FormatInteger(format string, n int) string {}
```

Wouldn't it be better to swap the arguments to `FormatRelTime` so it's consistent?